book/src/statements.md

```diff
@@ -1 +1,45 @@
 # Statements
+The claims asserted by a POD are referred to as its *statements*. These statements introduce values and express relations between them, where the values may or may not be part of the same POD. The mechanism for referring to values in arbitrary PODs is furnished by *anchored keys*.
+
+## Anchored keys
+Rather than dealing with just keys, we introduce the notion of an *anchored key*, which is a pair consisting of an origin specifier and a key, i.e.
+
+```
+type AnchoredKey = (Origin, Key)
+type Key = String
+```
+
+An *origin* is a triple consisting of a numeric identifier called the *origin ID*, a string called the *origin name* (omitted in the backend) and another numeric identifier called the *gadget ID*, which identifies the means by which the value corresponding to a given key is produced.
+
+The origin ID is defined to be 0 for 'no origin' and 1 for 'self origin', otherwise it is the content ID[^content-id] of the POD to which it refers. The origin name is not cryptographically significant and is merely a convenience for the frontend.
+
+The gadget ID takes on the values in the following table:
```

Comment:
I would like to discuss the idea of having Pod Class and Pod Type (Gadget Class and Gadget ID).
Gadget Class would be None, Signed, Main
Then within each class we have actual gadgets
- Signed: MockSignPod, SchnorrPod, FalconPod, Plonky2SignPod
- Main: MockMainPod, Plonky2Pod